streaming/index.js

```diff
@@ -329,29 +383,6 @@ const startServer = async () => {
     }
   };
 
-  /**
-   * @param {any} req
-   * @param {any} res
-   * @param {function(Error=): void} next
-   */
-  const setRequestId = (req, res, next) => {
-    req.requestId = uuid.v4();
```

Comment:
There are a bunch of typescript warnings due to dropping `requestId` in this pull request, but these are all resolved by the logging changes in #27828